WcaOnRails/app/models/competition.rb

```diff
@@ -240,7 +240,7 @@ def persisted_events_id
   # The only exception to this is within tests, in which case we actually don't want to run this validation.
   validate :schedule_must_match_rounds, if: :confirmed_at_changed?, on: :update
   def schedule_must_match_rounds
-    unless has_any_round_per_event? && schedule_includes_rounds?
+    unless has_any_round_per_event? && schedule_includes_rounds? && created_at < Date.new(2019, 12, 31)
```

Comment:
Did you mean to use greater than for this and the next one?

Reply:
No. This was supposed to make all the older competitions to be ignored in the error, which should work (since it says 'unless' and not 'if'). But now I realise that this addition will make every new competition give an error as they will never fulfil this unless clause. So there is the need for a different implementation.